core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -1119,6 +1131,21 @@ class ReplicaManager(val config: KafkaConfig,
         else
           Set.empty[Partition]
 
+        // in all of the partitions that the current broker is now a follower of
+        // add all the topics in a list as candidate to remove metrics
+        val topicCandidates = mutable.Set[String]()
```

Comment:
We can use some scala tricks here, rather than using a mutable.Set.

Reply:
Noted!